extensions/CheckUser/CheckUser_body.php

```diff
@@ -319,22 +326,22 @@ protected function noMatchesMessage( $userName, $checkLast = true ) {
 			if ( $user_id ) {
 				$revEdit = $dbr->selectField( 'revision',
```

Comment:
According to https://github.com/Wikia/app/blob/dev/includes/db/Database.php#L1201, it does.